src/index.test.ts

```diff
@@ -191,7 +198,7 @@ describe('basic retry and timeout usage', () => {
     const attempts = 3;
     jest.spyOn(operations, 'successFn');
 
-    const res = await go(operations.successFn, { attemptTimeoutMs: 5, retries: 2 });
+    const res = await go(operations.successFn, { attemptTimeoutMs: [5, 10, 15], retries: 2 });
```

Comment:
Why change this test?

Reply:
should have created a second one, instead of modifying an existing test. I was testing out if the retries worked as intended with an array